### PR TITLE
Support ElixirSense in multi folder workspaces

### PR DESCRIPTION
**Pull request by sadesyllas**

### CODE REVIEW

**Comment by fr1zle** [comment]
Will check this out during the week! Thanks for contributing

File: package.json [hunk 1]
```diff
@@ -6,13 +6,13 @@
         "Elixir",
         "syntax"
     ],
-    "version": "0.5.1",
+    "version": "0.6.0",
```
Comment (fr1zle) [inline package.json]: Can you keep the version the same? I usually only change it when I am building a new release.

**Comment by sadesyllas** [comment]
@fr1zle , I have changed the version back to `0.5.1`.

File: package.json (hunk 1 above)
Comment (sadesyllas) [reply package.json]: @fr1zle , I have changed the version back to 0.5.1.

**Comment by sadesyllas** [comment]
Thanks for merging this PR!

**Pull request merged**